crates/examples/examples/small_network_with_default.rs

```diff
@@ -1,14 +1,66 @@
+use std::time::Duration;
+
 use configuration::NetworkConfigBuilder;
+use orchestrator::{AddNodeOpts, Orchestrator};
+use provider::NativeProvider;
+use support::fs::local::LocalFileSystem;
 
-fn main() {
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let config = NetworkConfigBuilder::new()
         .with_relaychain(|r| {
             r.with_chain("rococo-local")
                 .with_default_command("polkadot")
                 .with_node(|node| node.with_name("alice"))
-                .with_node(|node| node.with_name("name"))
+                .with_node(|node| node.with_name("bob"))
+        })
+        .with_parachain(|p| {
+            p.with_id(100).cumulus_based(true).with_collator(|n| {
+                n.with_name("collator").with_command("polkadot-parachain")
+                //.with_command("adder-collator")
+            })
         })
-        .build();
+        .build()
+        .unwrap();
+
+    println!("{:?}", &config);
+
+    let fs = LocalFileSystem;
+    let provider = NativeProvider::new(fs.clone());
+    let orchestrator = Orchestrator::new(fs, provider);
+    let mut network = orchestrator.spawn(config).await?;
+    println!("🚀🚀🚀🚀 network deployed");
+    // add  a new node
+    let opts = AddNodeOpts {
+        rpc_port: Some(9444),
+        is_validator: true,
+        ..Default::default()
+    };
```

Comment:
I think we should define a impl AddNodeOpts to build is without having to call Default::default()